library/core/src/result.rs

```diff
@@ -1164,23 +1162,12 @@ impl<T: Deref, E> Result<T, E> {
     /// let y: Result<&str, &u32> = Err(&42);
     /// assert_eq!(x.as_deref(), y);
     /// ```
+    #[stable(feature = "inner_deref", since = "1.47.0")]
```

Comment:
About feature gate, do you in favor of renaming it to "result_as_deref" or so?

Reply:
I would just leave it I think.